tailwind.config.ts

```
@@ -8,65 +8,101 @@ const config: Config = {
       },
-      'asgardex-hyperlink': '#0070E0',
-      white: '#ffffff',
-      transparent: 'transparent'
-
-    },
-    backgroundImage: () => ({
-      'gradient-primary': 'linear-gradient(to right, #33FF99 , #00B4FF)'
-    })
+      backgroundImage: {
+        'gradient-primary': 'linear-gradient(9.34deg, #23DCC8 19.28%, #00CCFF 106.03%)',
+        'gradient-primary-dark': 'linear-gradient(9.34deg, #1bb2a3 19.28%, #00a3cc 106.03%)',
```

ESLint

[error] Unexpected trailing comma.

(@typescript-eslint/comma-dangle)